akka-docs/src/main/paradox/common/cluster.md

```diff
@@ -296,6 +296,7 @@ The `leader` has the following duties:
 
  * shifting members in and out of the cluster
     * joining -> up
+    * weakly up -> up (if enabled); *no convergence is required for this leader action to be performed*
```

Comment:
it's enabled by default in 2.5 so not sure if that () is needed

Reply:
Good catch, thanks! :)